spaceship/tasktiger.py

```diff
@@ -0,0 +1,35 @@
+
+import logging
+import pendulum
+import tasktiger
+
+from spaceship import logs
+from spaceship.redis import store
+
+class DBManager:
+  """A context manager that helps deal with DB sessions in tasks"""
+  def __init__(self):
+    self.log = logging.getLogger('tiger.dbmanager')
+
+    from spaceship.db import db
+    self.db = db
+
+  def __enter__(self):
+    self.log.debug("disposing of db engine")
+    self.db.engine.dispose()
+
+  def __exit__(self, ex_type, ex_value, ex_tb):
```

Comment:
should we also call db.session.remove()?

Reply:
(we think this is ok because the connection should get closed when the task subprocess exits)